coordinator/server/client_api.go

```diff
@@ -39,6 +39,9 @@ type StatusResp struct {
 }
 
 type ManifestSignatureResp struct {
+	// The manifest signature - signed by the root ECDSA key. Changes upon each call due to a random component.
```

Comment:
```suggestion
	// The manifest signature - signed by the root ECDSA key.
```
Should stay consistent now